File: sandbox.client/front/ts/model/ClientFilter.ts
export class ClientFilter {
    public firstname: string;
    public lastname: string;
    public patronymic: string;
    public orderBy: string;
    public sort: boolean;
    public page: number/*int*/;
    public pageTotal: number/*int*/;
    public recordSize: number/*int*/;
    public recordTotal: number/*int*/;

    constructor() {
        this.firstname = "";
        this.lastname = "";
        this.patronymic = "";
        this.orderBy = "";
        this.sort = true;
        this.page = 0;
      this.recordTotal = 0;
      this.recordSize = 20;
      this.pageTotal = 1

    }
    public assign(o: any): ClientFilter {
        this.firstname = o.firstname;
        this.lastname = o.lastname;
        this.patronymic = o.patronymic;
        this.orderBy = o.orderBy;
        this.sort = o.sort;
        this.page = o.page;
        this.pageTotal = o.pageTotal;
        this.recordSize = o.recordSize;
        this.recordTotal = o.recordTotal;
        return this;
    }
}

File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/model/ClientFilter.java
package kz.greetgo.sandbox.controller.model.model;



public class ClientFilter {
    public String firstname;
    public String lastname;
    public String patronymic;
    public String orderBy;
    public boolean sort;
    public int page;
    public int pageTotal;
    public int recordSize;
    public int recordTotal;

    public ClientFilter() {
        this.firstname = "";
        this.lastname = "";
        this.patronymic = "";
        this.orderBy = "";
        this.sort = true;
        this.page = 0;
        this.pageTotal = 0;
        this.recordSize = 10;
        this.recordTotal =0;
    }
}
